controller/sync_strats/sync_instant.go

```diff
@@ -191,7 +192,7 @@ func (strategy *InstantStrategy) RouterDisconnected(router *network.Router) {
 	log := pfxlog.Logger().WithField("sync_strategy", strategy.Type()).
```

Comment:
This is already fixed. Not sure why it doesn't show up that way.

Reply:
Weird. I didn't add this.

Reply:
Also this isn't what I named the PR.